neard/res/genesis_config.json

```diff
@@ -1,239 +1,239 @@
+          "send_not_sir": 4697339419375,
           "execution": 4697339419375
-        }, 
+        },
         "cost_per_byte": {
-          "send_sir": 59357464, 
-          "send_not_sir": 59357464, 
+          "send_sir": 59357464,
+          "send_not_sir": 59357464,
           "execution": 59357464
         }
-      }, 
+      },
       "action_creation_config": {
         "create_account_cost": {
-          "send_sir": 99607375000, 
-          "send_not_sir": 99607375000, 
+          "send_sir": 99607375000,
+          "send_not_sir": 99607375000,
           "execution": 99607375000
-        }, 
+        },
         "deploy_contract_cost": {
-          "send_sir": 184765750000, 
-          "send_not_sir": 184765750000, 
+          "send_sir": 184765750000,
+          "send_not_sir": 184765750000,
           "execution": 184765750000
-        }, 
+        },
         "deploy_contract_cost_per_byte": {
-          "send_sir": 6812999, 
-          "send_not_sir": 6812999, 
+          "send_sir": 6812999,
+          "send_not_sir": 6812999,
           "execution": 6812999
-        }, 
+        },
         "function_call_cost": {
-          "send_sir": 2319861500000, 
-          "send_not_sir": 2319861500000, 
+          "send_sir": 2319861500000,
+          "send_not_sir": 2319861500000,
           "execution": 2319861500000
-        }, 
+        },
         "function_call_cost_per_byte": {
-          "send_sir": 2235934, 
-          "send_not_sir": 2235934, 
+          "send_sir": 2235934,
+          "send_not_sir": 2235934,
           "execution": 2235934
-        }, 
+        },
         "transfer_cost": {
-          "send_sir": 115123062500, 
-          "send_not_sir": 115123062500, 
+          "send_sir": 115123062500,
+          "send_not_sir": 115123062500,
           "execution": 115123062500
-        }, 
+        },
         "stake_cost": {
-          "send_sir": 141715687500, 
-          "send_not_sir": 141715687500, 
+          "send_sir": 141715687500,
+          "send_not_sir": 141715687500,
           "execution": 102217625000
-        }, 
+        },
         "add_key_cost": {
           "full_access_cost": {
-            "send_sir": 101765125000, 
-            "send_not_sir": 101765125000, 
+            "send_sir": 101765125000,
+            "send_not_sir": 101765125000,
             "execution": 101765125000
-          }, 
+          },
           "function_call_cost": {
-            "send_sir": 102217625000, 
-            "send_not_sir": 102217625000, 
+            "send_sir": 102217625000,
+            "send_not_sir": 102217625000,
             "execution": 102217625000
-          }, 
+          },
           "function_call_cost_per_byte": {
-            "send_sir": 1925331, 
-            "send_not_sir": 1925331, 
+            "send_sir": 1925331,
+            "send_not_sir": 1925331,
             "execution": 1925331
           }
-        }, 
+        },
         "delete_key_cost": {
-          "send_sir": 94946625000, 
```

Comment:
Is change in costs pure trailing space removal?

Reply:
Yeah it is auto-generated by the script